crates/cxx-qt-lib/src/qml/qqmlapplicationengine.rs

```diff
@@ -6,6 +6,10 @@
 #[cxx_qt::bridge]
 mod ffi {
     unsafe extern "C++" {
+        include!("cxx-qt-lib/qbytearray.h");
+        type QByteArray = crate::QByteArray;
+        include!("cxx-qt-lib/qmap.h");
+        type QMap_QString_QVariant = crate::QMap<crate::QMapPair_QString_QVariant>;
```

Comment:
wonder if we can create a `QVariantMap` alias that works to make this a bit nicer, but it works :-)

Reply:
I think this is good enough for now, let's not introduce too many types around QMap, as we were thinking about refactoring that anyhow similar to https://github.com/KDAB/cxx-qt/issues/1138